.hydrogen/upgrade-2024.1.4-to-2024.4.0.md

```diff
@@ -1,13 +1,15 @@
 # Hydrogen upgrade guide: 2024.1.4 to 2024.4.0
 
-----
+---
```

Comment:
@blittle did you use prettier to fix formatting in this markdown file?
If so, maybe we should pass prettier when we generate this in the upgrade command @juanpprieto